bot-components/GitHub_queries.mli

```diff
@@ -0,0 +1,104 @@
+val send_graphql_query :
+     bot_info:Utils.bot_info
+  -> < parse: Yojson.Basic.t -> 'a
+     ; query: string
+     ; variables: Yojson.Basic.t
+     ; .. >
+  -> ('a, string) result Lwt.t
+
+type backport_info =
+  {backport_to: string; request_inclusion_column: int; backported_column: int}
+
+type full_backport_info =
+  {backport_info: backport_info list; rejected_milestone: string}
+
+val get_backport_info :
+  bot_info:Utils.bot_info -> string -> full_backport_info option
+
+type project_card =
+  { id: string
+  ; column: GitHub_GraphQL.project_column option
+  ; columns: GitHub_GraphQL.project_column list }
```

Comment:
We should not export anything from this internal module (`GitHub_GraphQL`). In fact, I introduced it only because I couldn't make Merlin understand the ppx but since it works fine now, there is no reason to keep this module anymore and we could move each GraphQL request next to the functions processing them.

Reply:
Why not make it a private module? It makes sense to separate the GraphQL requests from the functions processing. I'm also afraid that the file might get too long.

Reply:
That's fine by me.